molecule/default/tests/test_dashboard.py

```diff
@@ -4,22 +4,44 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
 @pytest.mark.jira('asc-222')
-def test_cinder_service(host):
```

Comment:
Is there a reason to delete this test?

Reply:
oh, I will revise the test_find_cinder_client() to become the test_cinder_service

Reply:
ok